src/mlpack/methods/rann/ra_search_impl.hpp

```diff
@@ -306,6 +306,31 @@ void RASearch<SortPolicy, MetricType, MatType, TreeType>::Train(
   }
 }
 
+//! Set the reference tree to a new reference tree.
+template<typename SortPolicy,
+         typename MetricType,
+         typename MatType,
+         template<typename TreeMetricType,
+                  typename TreeStatType,
+                  typename TreeMatType> class TreeType>
+void RASearch<SortPolicy, MetricType, MatType, TreeType>::Train(
+  Tree* referenceTree)
```

Comment:
I think this will fit all on one line in 80 characters, but if not then the indentation should be two tabs not one tabs on a wrapped line. :)